db/migrate/20210617140742_add_decision_date_to_project_dataset_level.rb

```diff
@@ -0,0 +1,6 @@
+# plan.io 25663 - add decision_date to project_dataset_level table
+class AddDecisionDateToProjectDatasetLevel < ActiveRecord::Migration[6.0]
+  def change
+    add_column :project_dataset_levels, :decision_date, :date
```

Comment:
`add_column :project_dataset_levels, :decided_at, :datetime`

That extra bit of granularity feels right in this context, even if we don't want to show it in the UI.